### DISCUSSION TITLE
settings/services doesn't seem to be able to display all the content on mobile

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#4463] [General]
![image](https://github.com/user-attachments/assets/903bba9f-512c-41fd-b362-5b0601fd15d8)

![image](https://github.com/user-attachments/assets/099a9bba-09d3-462a-aae2-18ed6a1bf030)

**Comment by graciousgrey** [comment]
Currently, tables on mobile screens show a reduced number of columns. The delete/edit function should be available when you turn the phone to horizontal mode. 

Would you prefer tables to be scrollable horizontally?
